test/fixtures/basic/package.json

```diff
@@ -4,6 +4,7 @@
     "run-series": "^1.1.1"
   },
   "devDependencies": {
+    "coffee-script": "^1.10.0",
```

Comment:
`coffee-script` happens to be a node package with zero dependencies, so it didn't make the tests time out.

In terms of making it programmatic, the only way I can think of doing that is copying the basic app fixture and inserting the dependency in there. Is there another way you were thinking of?